twitchio/websocket.py

```diff
@@ -294,7 +294,7 @@ async def _join_future_handle(self, fut: asyncio.Future, channel: str, timeout:
         try:
             await asyncio.wait_for(fut, timeout=timeout)
         except asyncio.TimeoutError:
-            log.error(f'The channel "{channel}" was unable to be joined. Check the channel is valid.')
+            self.dispatch("channel_join_failure", channel)
```

Comment:
Why remove the logging? This is still useful to be logged if people are not using the event for notification.